src/index.ts

```diff
@@ -165,9 +166,9 @@ class DefaultVisualizer {
      * @param  dataset visualizer input, containing a dataset and optional visualization settings
      * @return         Promise that resolves to a {@link DefaultVisualizer}
      */
-    public static load(config: DefaultConfig, dataset: Dataset): Promise<DefaultVisualizer> {
+    public static load(config: DefaultConfig, dataset: Dataset, warnings?: Warnings): Promise<DefaultVisualizer> {
```

Comment:
Why does this need to take a `warnings` instance? It could create one, since the instance will then be publicly accessible anyway

Reply:
I was thinking of creating this one step "up", in the app or the widget but I agree this is probably the right point to start the chain